client/src/components/MultiJurisdictionAudit/AASetup/Review/index.tsx

```diff
@@ -424,6 +424,7 @@ const Review: React.FC<IProps> = ({
               <FormButton
                 intent="primary"
                 disabled={
+                  sampleSizeOptions === null ||
```

Comment:
should be `(shouldShowSampleSizes && sampleSizeOptions === null)`

Reply:
I wondered about that, haha.